testsuite/features/step_definitions/common_steps.rb

```diff
@@ -57,8 +57,8 @@
 
   lines = interface.lines
   # selects only lines with IPv6 addresses and proceeds to form an array with only those addresses
-  ipv6_addresses_list = lines.grep(/2620:[:0-9a-f]*|fe80:[:0-9a-f]*/)
-  ipv6_addresses_list.map! { |ip_line| ip_line.slice(/2620:[:0-9a-f]*|fe80:[:0-9a-f]*/) }
+  ipv6_addresses_list = lines.grep(/2a07:[:0-9a-f]*|fe80:[:0-9a-f]*/)
```

Comment:
Maybe... we can think about using a regex that works for any MAC Address?
`^([0-9A-F]{2}[:-]){5}([0-9A-F]{2})$`

Or we want to keep this filtered by concrete prepend?

Reply:
We could indeed reduce the size of the prefix to digit `2` . This is the prefix for "global unicast" addresses. More exactly:
```
 ... (global unicast addresses) have a prefix of 2000::/3, (all the addresses that begin with binary 001).
```

Apart from that, it already works with any MAC address.

I'll do that change if prefix changes again (for example for the cloud).

Reply:
But... why we need to use one that has a prefix?
I miss something I guess. We are just checking that the MAC address is correct on that step right?

Reply:
I need to make sure I am controlling a real IPv6 address and not some random text.

We don't check a MAC address in that step, we check an IP address. It can include the MAC, or not, for example it can be loopback `::1` or a privacy address as in https://www.rfc-editor.org/rfc/rfc7217 .

Let's huddle.

Reply:
After discussion, let's check only the `2` (knowing that, strictly speaking, it could also be a `3`).

Reply:
Just to safe somewhere what discussed, we also discussed the possibility to use https://github.com/ruby/ipaddr which is part of the standard library, to validate the IPs by instantiating them.